opentelemetry-appender-tracing/README.md

```diff
@@ -32,3 +32,18 @@ management, and export of telemetry. A major goal of OpenTelemetry is that you
 can easily instrument your applications or systems, no matter their language,
 infrastructure, or runtime environment. Crucially, the storage and visualization
 of telemetry is intentionally left to other tools.
+
+*[Supported Rust Versions](#supported-rust-versions)*
```

Comment:
nit - do we need this line, given that the information is already clear and easy to find in the README?

Reply:
kept it for consistency, and also there'll be more content to be added to this readme in future.